config.py

```diff
@@ -844,7 +844,27 @@
             'Без извори',
         ],
         hidden_category = 'Скриени_категории',
-    )
+    ),
+
+    eo = dict(
+        lang_name = 'Esperanto',
+        lang_dir = 'ltr',
+        database = 'eowiki_p',
+        wikipedia_domain = 'eo.wikipedia.org',
+
+        beginners_link = 'https://eo.wikipedia.org/wiki/Vikipedio:Citi_fontojn',
+        beginners_link_title = 'Vikipedio:Citi fontojn',
```

Comment:
```suggestion
```

Reply:
`beginners_link` and `beginners_link_title` are actually obsolete now, can you please remove?